module/core/src/Application/Controller/Api/LanguageController.php

```diff
@@ -0,0 +1,236 @@
+     *     description="Order field",
+     * )
+     * @SWG\Parameter(
+     *     name="order",
+     *     in="query",
+     *     required=false,
+     *     type="string",
+     *     enum={"ASC","DESC"},
+     *     description="Order",
+     * )
+     * @SWG\Parameter(
+     *     name="show",
+     *     in="query",
+     *     required=false,
+     *     type="string",
+     *     enum={"COLUMN","DATA"},
+     *     description="Specify what response should containts"
+     * )
+     * @SWG\Parameter(
+     *     name="language",
+     *     in="path",
+     *     type="string",
+     *     required=true,
+     *     default="EN",
+     *     description="Language Code",
+     * )
+     * @SWG\Response(
+     *     response=200,
+     *     description="Returns import",
+     * )
+     * @SWG\Response(
+     *     response=404,
+     *     description="Not found",
+     * )
+     *
+     * @param Language $language
+     * @param Request  $request
+     *
+     * @return Response
+     */
+    public function getLanguages(Language $language, Request $request): Response
+    {
+        $configuration = new RequestGridConfiguration($request);
+
+        $dataSet = $this->query->getDataSet();
+        $result = $this->renderGrid($this->languageGrid, $configuration, $dataSet, $language);
+
+        return $this->createRestResponse($result);
+    }
+
+    /**
+     * @Route("/languages", methods={"PUT"})
+     *
+     * @SWG\Tag(name="Language")
+     *
+     * @SWG\Parameter(
+     *     name="language",
+     *     in="path",
+     *     type="string",
+     *     required=true,
+     *     default="EN",
+     *     description="Language Code",
+     * )
+     * @SWG\Parameter(
+     *     name="body",
+     *     in="body",
+     *     description="Category body",
+     *     required=true,
+     *     @SWG\Schema(ref="#/definitions/languages")
+     * )
+     * @SWG\Response(
+     *     response=201,
+     *     description="Update language",
+     * )
+     * @SWG\Response(
+     *     response=400,
+     *     description="Form validation error",
+     * )
+     *
+     * @param Request $request
+     *
+     * @return Response
+     * @throws \Exception
+     *
+     */
+    public function updateLanguage(Request $request): Response
+    {
+        try {
+            $model = new LanguageCollectionFormModel();
+            $form = $this->createForm(LanguageCollectionForm::class, $model, ['method' => 'PUT']);
+            $form->handleRequest($request);
+            if ($form->isSubmitted() && $form->isValid()) {
+                /** @var LanguageCollectionFormModel $data */
+                $data = $form->getData();
+                $languages = $data->collection->getValues();
+                foreach ($languages as $language) {
+                    $this->repository->save(Language::fromString($language->code), $language->active);
+                }
+
+                return $this->createRestResponse(['message' => "Updated"]);
```

Comment:
Change it to ``return $this->createRestResponse();``. You don't need to send message

Reply:
Once I changed 
" "code": 500,
  "message": "Too few arguments to function Ergonode\\Core\\Application\\Controller\\AbstractApiController::createRestResponse(), 0 passed in /home/mkolanko/www/local/ergonode-wiewiurd-from-github/module/core/src/Application/Controller/Api/LanguageController.php on line 242 and at least 1 expected",
  "trace": ["

Reply:
Add empty array :)